src/mpyl/cli/build.py

```diff
@@ -262,14 +262,34 @@ def by_choice(choice: Choice):
         return release.tag_name
 
 
-def ask_for_input(ctx, _param, value) -> Optional[str]:
+def select_target():
+    return questionary.select(
+        "Which environment do you want to deploy to?",
+        show_selected=True,
+        choices=[
+            Choice(title="Acceptance", value=Target.ACCEPTANCE.value),
+            Choice(title="PullRequestBase", value=Target.PULL_REQUEST_BASE.value),
+            Choice(title="Production", value=Target.PRODUCTION.value),
+        ],
+    ).ask()
+
+
+def ask_for_tag_input(ctx, _param, value) -> Optional[str]:
     if value == "not_set":
         return None
     if value == "prompt":
         return select_tag(ctx)
     return value
 
 
+def ask_for_target_input(ctx, _param, value) -> Optional[str]:
+    if value == "not_set":
+        return None
+    if value == "prompt":
+        return select_target()
+    return value
```

Comment:
maybe you could pass the function in as a parameter.

Reply:
I tried, but I think callbacks have a set signature in click